test/integration/cli/config.test.js

```diff
@@ -17,11 +17,14 @@ describe('cli --webpack-config', function () {
     });
   });
 
-  it('throws when not found', function (done) {
+  it('throws when not found', async function () {
     const configNotFound = 'xxxxxxx.js';
-    exec(`node ${binPath} --webpack-config ${configNotFound} "${testSimple}"`, (err) => {
-      assert.include(err.message, `Webpack config could not be found: ${configNotFound}`);
-      done();
+    const command = `node ${binPath} --webpack-config ${configNotFound} "${testSimple}"`;
+    await new Promise((res) => {
```

Comment:
Is async await is really needed in this test, why?

Reply:
Good catch, that was an artifact from a misunderstanding I had about how to catch the error in the test. This can just rely on the regular callback in `exec`. I've fixed this in a subsequent commit. Also changed the variable name from `stdout` to `output` to match other specs.